benchmark-service/benchmark/benchmark.py

```diff
@@ -36,6 +36,9 @@
 
 benchmark_data = None
 
+with open(os.environ.get('HAIL_CI_OAUTH_TOKEN', 'oauth-token/oauth-token'), 'r') as f:
```

Comment:
Why do you need this?

Reply:
for the github client

Reply:
I understand that, but why did you not have it before and it seemed to be working. Oh. I see. Why did you move it?

Reply:
I was getting some weird session closed errors, then after I moved it they went away

Reply:
ok. that's what CI does.